packages/core/src/storage.ts

```diff
@@ -13,10 +13,23 @@
 // limitations under the License.
 //
 
-import type { Class, Doc, Ref } from './classes'
+import type { Class, Doc, Obj, Ref } from './classes'
 import type { Tx } from './tx'
 
-export type DocumentQuery<T extends Doc> = Partial<T>
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type QuerySelector<T> = {
+  $gt?: T
```

Comment:
Why we need any other then $in?

Reply:
I don't know) Some overengineering, because I think `operators like $in` is more then only `$in`)

Reply:
I can delete it, and add when we necessary